src/services/cgnService.ts

```diff
@@ -69,4 +80,51 @@ export default class CgnService {
         }
       });
     });
+
+  /**
+   * Get the current CGN Status related to the user.
+   */
+  public readonly startCgnActivation = (
+    user: User
+  ): Promise<
+    // tslint:disable-next-line: max-union-size
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseErrorForbiddenNotAuthorized
+    | IResponseErrorConflict
+    | IResponseSuccessRedirectToResource<InstanceId, InstanceId>
+    | IResponseSuccessAccepted
+  > =>
+    withCatchAsInternalError(async () => {
+      const validated = await this.cgnApiClient.startCgnActivation({
+        fiscalcode: user.fiscal_code
+      });
+
+      return withValidatedOrInternalError(validated, response => {
+        switch (response.status) {
+          case 201:
+            return ResponseSuccessRedirectToResource(
+              response.value,
+              fromNullable(response.headers.Location).getOrElse(
+                "/api/v1/cgn/activation"
+              ),
+              response.value
+            );
+          case 202:
+            return ResponseSuccessAccepted();
+          case 401:
+            return ResponseErrorUnexpectedAuthProblem();
```

Comment:
What scenario can cause this?

Reply:
Only a misconfiguration ( missing or wrong API key)